docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -207,15 +264,21 @@ Once your pki provider is configured we will create the provisioner configuratio
 
 Here is an example on how to configure an EAP-TLS connection for Windows/Android/MacOS/iOS
 
-image::docs/images/eaptls-example.png [scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
+image::docs/images/eaptls-example.png[scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
 
-+ explanation of screenshot
+* Provisioning ID: Name of your provisioner in PacketFence and of the profile on MacOS/iOS when installed
```

Comment:
I thought that this was changed to use the description when installing the profile ?